src/Confluent.Kafka/AdminClient.cs

```diff
@@ -207,6 +207,23 @@ private List<DeleteAclsReport> extractDeleteAclsReports(IntPtr resultPtr)
             }).ToList();
         }
 
+        private List<DeleteConsumerGroupOffsetsReport> extractDeleteConsumerGroupOffsetsReports(IntPtr resultPtr)
+        {
+            IntPtr groupsOffsetsResultPtr = Librdkafka.DeleteConsumerGroupOffsets_result_groups(resultPtr, out UIntPtr resultCountPtr);
+            int groupsOffsetsResultCount = (int)resultCountPtr;
+            IntPtr[] groupsOffsetsResultArr = new IntPtr[groupsOffsetsResultCount];
+            Marshal.Copy(groupsOffsetsResultPtr, groupsOffsetsResultArr, 0, groupsOffsetsResultCount);
+
+            return groupsOffsetsResultArr.Select(groupOffsetsResultPtr => new DeleteConsumerGroupOffsetsReport
```

Comment:
follow the coding convention in other methods, put the new DeleteConsumeGroupOffsetsReport on a new line.

Reply:
oh, ok, i see i did it like this in one place above too, same comment to historic Matt :-).
no need to change ...